thrall/app/lib/ElasticSearch.scala

```diff
@@ -74,7 +74,10 @@ object ElasticSearch extends ElasticSearchClient {
           missingOrEmptyFilter(s"identifiers.$persistenceIdentifier"),
           boolFilter.should(
             missingOrEmptyFilter("userMetadata.archived"),
-            boolFilter.must(termFilter("userMetadata.archived", false))
+            boolFilter.must(termFilter("userMetadata.archived", false)),
+            boolFilter.mustNot(termFilter("usageRights.category", "staff-photographer")),
```

Comment:
We have [these helpers](https://github.com/guardian/grid/blob/master/media-api/app/lib/elasticsearch/ImageFields.scala) to try mitigate string usage.